docs/debian-conf.d/router/250_vexim_virtual_domains

```diff
@@ -65,8 +65,9 @@ virtual_domains:
                                and users.on_spamassassin = 0 \
                                and users.domain_id=domains.domain_id}{$value}fail}}} \
                            } {X-Spam-Score:X-Spam-Report} }
-  local_part_suffix = -*
-  local_part_suffix_optional
+  # optional!
+  #local_part_suffix = -*
+  #local_part_suffix_optional
```

Comment:
I think these should be enabled by default, cause they are enabled in docs/configure.
On the other hand, IF these lines aren't specific to vexim routers, they could probably be removed altogether. However, it seems they are specific to our routers (aren't they?), so they should probably stay.